From b790f87d017c5dbc6bcebacc5ab283c05e379842 Mon Sep 17 00:00:00 2001
From: "Valery V. Vorotyntsev" <valery_vorotyntsev@xyratex.com>
Date: Fri, 28 Jun 2013 14:25:19 +0300
Subject: [PATCH 250/290] be: resolve name conflict (altogether-mode)

be/tx.c: rename _state_set() to _tx_state_set().
static _state_set() function was defined in two places (conf/confc.c
and be/tx.c), preventing compilation in --enable-altogether-mode from
succeeding.
---
 be/tx.c | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/be/tx.c b/be/tx.c
index 3b9cc21..1e0a7f4 100644
--- a/be/tx.c
+++ b/be/tx.c
@@ -556,7 +556,8 @@ M0_INTERNAL void tx_state_set(struct m0_be_tx *tx, enum m0_be_tx_state state)
 	M0_POST(m0_be__tx_invariant(tx));
 }
 
-static void _state_set(struct m0_sm_group *grp M0_UNUSED, struct m0_sm_ast *ast)
+static void
+_tx_state_set(struct m0_sm_group *grp M0_UNUSED, struct m0_sm_ast *ast)
 {
 	enum m0_be_tx_state state = *(enum m0_be_tx_state *)ast->sa_datum;
 	M0_ENTRY("state=%u", state);
@@ -594,7 +595,7 @@ m0_be__tx_state_post(struct m0_be_tx *tx, enum m0_be_tx_state state)
 	M0_PRE(M0_IN(state, (M0_BTS_SUBMITTED, M0_BTS_LOGGED, M0_BTS_PLACED,
 			     M0_BTS_STABLE)));
 
-	tx->t_ast.sa_cb    = _state_set;
+	tx->t_ast.sa_cb    = _tx_state_set;
 	tx->t_ast.sa_datum = &tx->t_ast_datum;
 	tx->t_ast_datum    = state;
 	m0_sm_ast_post(tx->t_sm.sm_grp, &tx->t_ast);
-- 
1.8.3.2

